src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
+  background-color: #ddd;
+  color: black;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
```

Comment:
default value - no need to set it (the same as line-height)

Reply:
Done! For exception font-size.